### PR TITLE
feat(deleteFolderFromBrowse): Delete folder from browse

### PR DESCRIPTION
**Pull request by simran142002**
## Description

feature(Enhancement) to delete folder from browse

### Changes

Called the delete method from admin folder delete in ui browse.php to enable deleting of folder

## How to test

It can be tested from browse page

Closing #2394 

[Screencast from 23-03-23 04:01:07 PM IST.webm](https://user-images.githubusercontent.com/95395832/227179630-ff8aacef-2fa4-443b-b2b6-99a927114d51.webm)

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2395"></a>

### CODE REVIEW

**Comment by shaheemazmalmmd** [comment]
@simran142002 Can you please add the confirmation and reload it to show jobs page as you did it in your upload delete PR ??

**Comment by shaheemazmalmmd** [comment]
> > @simran142002 Can you please add the confirmation and reload it to show jobs page as you did it in your upload delete PR ??
> 
> Sure sir I will make the changes!.. but just one thing do we need confirmation here because its written that its for deleting the folder beside the button?

It is required, because all the data/components inside that folder will be deleted also we can prevent accidental clicks.

**Comment by shaheemazmalmmd** [comment]
@simran142002 : Please fix the alignments.

```
FILE: /home/runner/work/fossology/fossology/src/www/ui/ui-browse.php
----------------------------------------------------------------------
FOUND 15 ERRORS AFFECTING 14 LINES
----------------------------------------------------------------------
 171 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 172 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 173 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 174 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 175 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 176 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 177 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 178 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 179 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 180 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 181 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 186 | ERROR | [x] Line indented incorrectly; expected at least 6
     |       |     spaces, found 4
 [219](https://github.com/fossology/fossology/actions/runs/4499587776/jobs/7917639291?pr=2395#step:4:220) | ERROR | [x] Whitespace found at end of line
 248 | ERROR | [x] Line indented incorrectly; expected 4 spaces,
     |       |     found 5
 248 | ERROR | [x] Closing brace indented incorrectly; expected 4
     |       |     spaces, found 5
----------------------------------------------------------------------
PHPCBF CAN FIX THE 15 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

File: src/www/ui/ui-browse.php [hunk 1]
```diff
@@ -167,6 +167,24 @@ function ShowItem($Upload, $Item, $Show, $Folder, $uploadtree_tablename)
...
+    $deleteFolder.= "<P>$text\n";
+    $deleteFolder.= "<select name='deleteFolder' class='ui-render-select2'>\n";
+    $text = _("select folder");
+    $deleteFolder.= "<option value='' disabled selected>$text</option>\n";
+    $deleteFolder.= FolderListOption(-1, 0, 1, -1, true);
+    $deleteFolder.= "</select>";
+    $text = _("Delete");
+    $deleteFolder.= "&nbsp<input type='submit' onclick='myFunction()' value='$text' class='btn btn-outline-danger' style='font-size: 10px;'><script>
+    function myFunction() {
+      alert('Are you sure you want to delete this upload ?');
```
Comment (shaheemazmalmmd) [inline src/www/ui/ui-browse.php:183]: All uploads inside this folder will also get deleted. Are you sure you want to delete this folder ?

**Comment by simran142002** [comment]
Was unable to resolve the conflict issues properly so raising a new pr

**Pull request closed without merging**